.github/workflows/nightly-tests.yml

```diff
@@ -69,8 +69,10 @@ jobs:
       run: rustup toolchain install 1.81-x86_64-unknown-linux-gnu
     - name: Install nightly
       run: rustup toolchain install nightly-2024-08-01-x86_64-unknown-linux-gnu
-    - name: Install stdlib
+    - name: Install std source
       run: rustup component add rust-src --toolchain nightly-2024-08-01-x86_64-unknown-linux-gnu
+    - name: Install riscv target
+      run: rustup target add riscv32imac-unknown-none-elf --toolchain nightly-2024-08-01-x86_64-unknown-linux-gnu
```

Comment:
We built the core libraries from source, since we needed the .s files. Now we can use the pre-built.

Then you ask: "Then why do we still need the line above, that installs the rust source?"

Because the risc0 target we use for std support is a lower tier and doesn't have pre-builds, so we still need to build `std` from source when using it.